packages/design-system/src/types/Table/Table.d.ts

```diff
@@ -16,9 +16,9 @@ export interface TableProps {
    */
   className?: string;
   /**
-   * Applies the dense variation of the table.
+   * Applies the compact variation of the table.
    */
-  dense?: boolean,
+  compact?: boolean,
```

Comment:
Similar to other props that are renamed, can we add a warning for folks using the `dense` prop? Search for `console.warn` for examples